cms/tests/test_titlepath.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+from cms.api import create_page, create_title, publish_pages
+from cms.test_utils.testcases import (CMSTestCase)
+
+
+class TitlePathTestCase(CMSTestCase):
+
+    def test0001_translated_subpage_title_path(self):
+        """
+        Test the path when parent page is created after child translation 
+        """
+
+        p_1 = create_page('en-1', "nav_playground.html", 'en',
+                          slug = 'en-1', published=True)
+        p_1_1 = create_page('en-1-1', "nav_playground.html", 'en',
+                            slug = 'en-1-1', parent=p_1, published=True)
+
+        create_title('de', 'de-1-1', p_1_1, slug='de-1-1')
+
+        # Parent 'de' title created after de-1-1 translation
+        create_title('de', 'de-1', p_1, slug='de-1')
+
+        p_1.publish('de')
+        p_1_1.publish('de')
+
+        response = self.client.get('/de/de-1/')
+        self.assertEqual(response.status_code, 200)
+
+        response = self.client.get('/de/en-1/de-1-1/')
+        if response.status_code == 200:
+            print('\n********** Unexpected response 200 for /de/en-1/de-1-1/')
```

Comment:
We shouldn't print in the test suite like this, if you really have to use the logger. I would advise not printing at all, only output if an error exists. 
```
# We should get a 404 not a successful 200 response, because the slug is wrong: /de/en-1/de-1-1/
self.assertEqual(response.status_code, 200)

Reply:
Yes I do agree. It was just to trace the bug.

Reply:
I have removed unnecessary test code lines